subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/NamedDomainObjectCollectionExtensions.kt

```diff
@@ -221,12 +221,21 @@ private constructor(
         fun <T> of(delegate: T) =
             ExistingDomainObjectDelegate(delegate)
     }
+
+    /**
+     * Gets the delegate value.
+     *
+     * @since 6.5
+     */
+    operator fun getValue(receiver: Any?, property: KProperty<*>): T =
```

Comment:
❌ The existing extension is just missing an `out` constraint on the generic argument, should be `ExistingDomainObjectDelegate<out T>...`

Reply:
Thanks! Fixed